android/java/res/layout/activity_select_purchase_method.xml

```diff
@@ -0,0 +1,249 @@
+                    android:textSize="20sp"
+                    android:textColor="@color/wallet_text_color"
+                    android:gravity="center_vertical"
+                    android:layout_marginEnd="20dp"
+                    android:text="@string/wallet_buy_mainnet_button_text"/>
+
+            </LinearLayout>
+
+        </androidx.appcompat.widget.Toolbar>
+
+        <TextView
+            android:id="@+id/select_options_text"
+            style="@style/BraveWalletTextView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="14sp"
+            android:layout_marginStart="16dp"
+            android:layout_marginEnd="16dp"
+            android:layout_marginBottom="8dp"
+            android:textColor="@color/wallet_secondary_text_color"
+            android:text="@string/brave_wallet_select_following_options"/>
+
+    <LinearLayout
+        android:id="@+id/ramp_network_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingTop="8dp"
+        android:paddingBottom="8dp"
+        android:paddingStart="14dp"
+        android:paddingEnd="14dp"
+        android:layout_marginStart="14dp"
+        android:layout_marginEnd="14dp"
+        android:layout_marginBottom="8dp"
+        android:background="@drawable/rounded_filled_bg_radius_12"
+        android:orientation="horizontal"
+        android:baselineAligned="false"
+        android:visibility="gone">
+
+        <ImageView
+            android:layout_width="74dp"
+            android:layout_height="48dp"
+            android:layout_gravity="top"
+            android:paddingTop="8dp"
+            android:paddingEnd="14dp"
+            app:srcCompat="@drawable/ramp_logo"
+            android:contentDescription="@null" />
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <TextView
+                style="@style/BraveWalletTextViewTitle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:gravity="start"
+                android:paddingVertical="8dp"
+                android:text="@string/brave_wallet_ramp_network_title"
+                android:textColor="@color/wallet_text_color"
+                android:textSize="17sp" />
+
+            <TextView
+                style="@style/BraveWalletTextView"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:gravity="start"
+                android:paddingVertical="8dp"
+                android:text="@string/brave_wallet_buy_crypto_visa_mastercard"
+                android:textColor="@color/wallet_text_color"
```

Comment:
any reason why `android:textColor="wallet_text_color` is needed for all the textviews? seems like a CP error?

Reply:
Because `BraveWalletTextView` has basic colour (per figma), size and dimension support 

I have added some styles to reuse and feel free to add more if something can be reused. We should always prefer to use these styles. These style attributes can be overridden in XML if we need to do a single/small change etc.

https://github.com/brave/brave-core/blob/master/android/java/res/values/brave_styles.xml#L64-L70